be1-go/cli/mod.go

```diff
@@ -19,23 +19,30 @@ import (
 	"golang.org/x/xerrors"
 )
 
+const witness string = "witness"
+const organizer string = "organizer"
+
 // Serve parses the CLI arguments and spawns a hub and a websocket server for
-// the organizer.
-func Serve(cliCtx *cli.Context) error {
+// the organizer or the witness
+func Serve(cliCtx *cli.Context, user string) error {
 	log := be1_go.Logger
 
-	// get command line args which specify public key, port to use for clients,
-	// witnesses and witness' address
+	if user != organizer && user != witness {
+		return xerrors.Errorf("unrecognized user, should be \"organizer\" or \"witness\"")
+	}
+
+	// get command line args which specify public key, port to use for clients
+	// and witnesses, witness' address
 	clientPort := cliCtx.Int("client-port")
 	witnessPort := cliCtx.Int("witness-port")
 	if clientPort == witnessPort {
 		return xerrors.Errorf("client and witness ports must be different")
 	}
-	witness := cliCtx.StringSlice("other-witness")
+	otherWitness := cliCtx.StringSlice("other-witness")
```

Comment:
I think we would need a variable to add one (or more) organizer too

Reply:
I will leave it for some future work as the system does not support multiple organisers yet